app/components/modelling/galaxyScore.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+// grabs the model score (calculated in modelCanvas) and displays it with a
+// progress bar
+const ModelScore = (props) => {
+  if (props.workflow && props.workflow.configuration &&
+    props.workflow.configuration.metadata &&
+    props.workflow.configuration.metadata.modelScore) {
+    let s = props.workflow.configuration.metadata.modelScore;
+    if (s < 80) {
+      s = s.toFixed(2);
+    } else if (s < 95) {
+      s = (s).toFixed(3);
+    } else {
+      s = (s).toFixed(4);
+    }
+    return (
+      <div className="answer undefined">
+        <div className="answer-button">
+          <div className="answer-button-label">Score: {s}</div>
```

Comment:
Why is this using a `button` class if it isn't a button? Will this give the visual impression that it is a clickable element?

Reply:
I have no idea where these classes materialised from. Don't think they have any impact whatsoever